astropy/units/core.py

```diff
@@ -261,6 +272,48 @@ def add_enabled_equivalencies(self, equivalencies):
         equivalencies = _normalize_equivalencies(equivalencies)
         self._equivalencies |= set(equivalencies)
 
+    def set_enabled_aliases(self, aliases):
+        """
+        Set aliases for units, e.g., to handle alternate spellings.
+
+        Parameters
+        ----------
+        aliases : dict of str, Unit
+            The aliases to set. The keys must be the string aliases, and values
+            must be the `astropy.units.Unit` that the alias will be mapped to.
+
+        Raises
+        ------
+        ValueError
+            If the alias already defines a different unit.
```

Comment:
Does this get raised? Maybe a copy-paste error from `add_`?

Reply:
It can get raised if one is trying to define an alias using a string that already exists as a regular unit

Reply:
Ah, right